src/api.js

```diff
@@ -1,26 +1,25 @@
-const API_URL = 'https://pokeapi.co/api/v2/';
+const BASE_URL = 'https://pokeapi.co/api/v2/';
 
-// make an API call to 'PokeAPI'
-const apiCall = async ( endpoint ) => {
-	return fetch( API_URL + endpoint )
-		.then( ( res ) => res.json() );
+// Make a GET request to 'PokeAPI'.
+const get = async ( endpoint ) => {
+	return fetch( BASE_URL + endpoint ).then( ( res ) => res.json() );
 };
 
-// fetch all pokemons
+// Fetch all pokemons.
 export const fetchPokemons = async ( limit = 151, offset = 0 ) => {
-	return apiCall( `pokemon?limit=${ limit }&offset=${ offset }` );
+	return get( `pokemon?limit=${ limit }&offset=${ offset }` );
```

Comment:
Why this function is async?
151 hardcoded?

Reply:
Removed